src/pudl/extract/xbrl.py

```diff
@@ -85,12 +86,17 @@ def xbrl2sqlite(context) -> None:
             logger.info(f"Dataset ferc{form}_xbrl is disabled, skipping")
             continue
 
+        sql_path = Path(urlparse(PudlPaths().sqlite_db(f"ferc{form.value}_xbrl")).path)
+
+        if clobber:
+            sql_path.unlink(missing_ok=True)
+
```

Comment:
okay so this is now the guy that removes the db and we always run `clobber=True` in pudl when we call the extractor's `run_main` bc each run is per year. cool cool!